mne/channels/tests/test_montage.py

```diff
@@ -959,14 +949,38 @@ def test_set_montage():
     raw = read_raw_fif(fif_fname)
     orig_pos = np.array([ch['loc'][:3] for ch in raw.info['chs']
                          if ch['ch_name'].startswith('EEG')])
-    raw.set_montage('mgh60')  # test loading with string argument
+    with pytest.deprecated_call():
+        raw.set_montage('mgh60')  # test loading with string argument
```

Comment:
this should still work but via the DigMontage route

Reply:
#6765